### DISCUSSION TITLE
Full Preact Tutorial Bug

### DISCUSSION THREAD

**Discussion by mliudev** [preactjs/preact-www#1007] [Q&A] [answered]
The full preact tutorial page seems to have a bug where the page renderer isn't working. I have attached a screenshot. I tried this in the following browsers:

- Firefox 116.0.3
- Chrome 116.0.5845.110 (Official Build) (arm64)
![preact_tutorial_bug](https://github.com/preactjs/preact/assets/11674155/12673345-6c82-49b0-a9be-2d01e3a46f01)

**Comment by rschristian** [comment, marked as answer]
Fixed in #1009
